students/seungho-shin/user/views.py

```diff
@@ -0,0 +1,35 @@
+import json
+
+from django.http            import JsonResponse
+from django.views           import View
+from django.core.exceptions import ValidationError
+from user.models            import User
+from user.validators        import validate_email
+
+class SignupView(View):
+    def post(self, request):
+        data    = json.loads(request.body)
+        try:
+            if not validate_email(data['email']):
+                return JsonResponse({'MASSAGE':'INVALID EMAIL'}, status=400)    
+            
+            if not len(data['password']) >= 8:
+                return JsonResponse({'MASSAGE':'INVALID PASSWORD'}, status=400)
+
+            if not User.objects.filter(email = data['email']):
```

Comment:
```suggestion
            if not User.objects.filter(email = data['email']).exists():
```
.exists()을 사용하면 데이터를 가져오지 않고 존재 유무만 판단할 수 있어 조금 더 효율적이고 가독성이 높습니다.